Games/Chess/main.cpp

```diff
@@ -65,6 +67,16 @@ int main(int argc, char** argv)
         return 1;
     }
 
+    if (unveil("/home/anon/", "wcb") < 0) {
+        perror("unveil");
+        return 1;
+    }
+
+    if (unveil("/etc/passwd", "rb") < 0) {
+        perror("unveil");
+        return 1;
+    }
```

Comment:
`unevil` Browse `b` should not be necessary for `/etc/passwd` :

```suggestion
    if (unveil("/etc/passwd", "r") < 0) {
        perror("unveil");
        return 1;
    }
```